app.js

```diff
@@ -50,7 +65,7 @@ const renderTodos = () => {
     todoList$.innerHTML = null;
 
     todos.forEach(todo => {
-        todoList$.appendChild(createTodoElement(todo));
+        todoList$.appendChild(onCreateElement(todo));
```

Comment:
The `on` prefix in a function suggests that, the function is supposed to be called **on** an event. So generally when we attach event listeners to elements such as **buttons** and **inputs**, we prefix the function with **on** keyword. If a function isn't used as event listener, we can omit the **on** prefix and name it normally.

```suggestion
        todoList$.appendChild(createElement(todo));
```